src/store/user/saga.ts

```diff
@@ -25,15 +25,17 @@ function* signUpAsync({ payload }: any) {
 
 function* signInAsync({ payload }: any) {
   try {
+
     yield put(actionObject(SHOW_LOADER, true))
 
     let response = yield call(GraphQlClient, loginUser(payload))
     response = validateFetch(response)
 
     yield put(actionObject(SIGN_UP_ASYNC, {
       user: {
-        ...response.login.user,
-        authToken: response.login.authToken
+        ...{ ...response.login.user, ...response.login.customer },
```

Comment:
This is equal to ...response.login.user, ...response.login.customer.

Use ...{ ...rest } is unnecessary